### DISCUSSION TITLE
Domain falsely blocked (auriswap.com)

### DISCUSSION THREAD

**Discussion by myounisshafi** [phantom/docs#239] [Help & Support] [answered]
### Summary

Hi. auriswap.com is the official website of the Auriswap Dex, the first dex on Vanar Chain (see URL in bio: https://twitter.com/AuriSwapDEX). It's currently being blocked by Phantom. 
Also, we checked the blocklist and we are not present there, so not sure why is blocking us: https://github.com/phantom/blocklist/blob/master/blocklist.yaml
Please suggest solution or fix it. 
We saw a similar issue Resolved by Maintainer.(https://github.com/orgs/phantom/discussions/231)

### Example

https://auriswap.com/

### Steps to Reproduce

Installed the Phantom extension and site is getting blocked. Tried both crome and brave.
<img width="1544" alt="Screen Shot 2024-05-29 at 4 39 48 PM" src="https://github.com/phantom/docs/assets/6582204/516e0084-7b22-44e1-956d-0391eef595ea">
 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @myounisshafi,

We are looking into this. Thank you for flagging.

**Reply by adamdelphantom** [reply, marked as answer]
@myounisshafi This should be resolved now.

**Reply by myounisshafi** [reply]
Yes. Thank You. Its working now.
